internal/k8s/client.go

```diff
@@ -202,6 +202,8 @@ func (cli *Client) CreateCluster(ctx context.Context, namespace string, cluster
 		return "", err
 	}
 
+	slog.Info("creating cluster", "namespace", namespace, "cluster", unstructuredCluster)
```

Comment:
I wonder if this is need or maybe a different level? not sure how look like the unstructuredCluster in the output